components/expenses/CreateExpenseForm.js

```diff
@@ -65,21 +66,26 @@ class CreateExpenseForm extends React.Component {
         id: 'expense.error.attachmentMissing',
         defaultMessage: 'Missing attachment',
       },
+      'error.expenseTypeMissing': {
+        id: 'expense.error.expenseTypeMissing',
+        defaultMessage: 'Please pick the type of this expense',
+      },
     });
 
     this.categoriesOptions = categories(props.collective.slug).map(category => {
       return { [category]: category };
     });
 
-    this.expenseTypes = Object.keys(expenseTypes).map(t => {
-      return { [t]: titleCase(t) };
+    this.expenseTypes = Object.entries(expenseTypes).map(t => {
+      const [key, value] = t;
```

Comment:
```suggestion
```

Reply:
@anshumanv sorry, for my suggestion to work, there was a second line required. Feel free to squash in 1 commit for a cleaner git history.

Reply:
Ahh my bad, fixed @znarf 

Lessons: Don't commit from web in haste ✨ 

Squashed 👍